CHANGELOG.md

```diff
@@ -7,6 +8,51 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.22.0a3] - 2025-01-10
+
+### Added
+
+- Propagate provided dag_display_name to built dag by @pankajkoti in #326
+- Add incipient documentation tooling by @tatiana in #328
+- Support loading `default_args` from shared `defaults.yml` by @pankajastro in #330
+- Add security policy by @tatiana in #339
+- Add Robust Support for Callbacks at Task and TaskGroup Level by @@jroach-astronomer in #322
+- Support `ExternalTaskSensor` `execution_date_fn` and `execution_delta` by @tatiana in #354
+- Refactor and add support for schedule conditions in DAG configuration by @ErickSeo in #320
```

Comment:
Need to rebase branch after merging this PR

Reply:
Done